src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -27,39 +32,70 @@
  */
 public class ConcurDetailLineGroupForCollector {
 
+    protected static final String ORPHANED_CASH_ADVANCE_MESSAGE_FRAGMENT = "Cash Advance did not have a corresponding entry from the Request Extract.";

```

Comment:
Since it's only a couple of them I think it might be better to move them now. Thank you.